containers/ecr-viewer/e2e/non-integrated/auth.spec.ts

```diff
@@ -36,7 +36,7 @@ test.describe("keycloak", () => {
         "The eCR Viewer couldn't retrieve the associated eCR file",
       ),
```

Comment:
if the non-integrated is still having some flakiness should we add a sleep or higher timeout or something?